lib/srv/desktop/rdp/rdpclient/src/client.rs

```diff
@@ -1169,6 +1169,7 @@ fn create_config(params: &ConnectParams, pin: String) -> Config {
         autologon: true,
         pointer_software_rendering: false,
         performance_flags: PerformanceFlags::default()
+            | PerformanceFlags::DISABLE_CURSOR_SHADOW // this is required for pointer to work correctly in Windows 2019 
```

Comment:
Wireshark is showing me a value of `0x00000086` here with FreeRDP, which is `PerformanceFlags(DISABLE_FULLWINDOWDRAG | DISABLE_MENUANIMATIONS | ENABLE_FONT_SMOOTHING)`. 

Does this issue reproduce with FreeRDP?